build-iam.yaml

```diff
@@ -1,21 +1,25 @@
 platform: linux
 inputs:
   - name: source
+  - name: common-tasks
 outputs:
   - name: source-out
 image_resource:
   type: docker-image
   source:
-    repository: python
-    tag: alpine
+    repository: quay.io/mojanalytics/alpine-python-git-jq
 run:
   path: sh
   args:
   - -ec
   - |
     cp -R ./source/. ./source-out/
-    if [ -f ./source/iam_config.yaml ] && [ ! -f ./source/iam_policy.json ]; then
+    if [[ -f ./source/iam_config.yaml ]] && [[ ! -f ./source/iam_policy.json ]]; then
       echo 'Building iam_policy.json from iam_config.yaml'
       pip install iam_builder
-      iam_builder -c ./source/iam_config.yaml -o ./source-out/iam_policy.json
+      iam_builder -c ./source/iam_config.yaml -o ./source/iam_policy.json
     fi
+    export ROLE_NAME=$(cat source/deploy.json | jq -r '.role_name')
+    deny=$(sed 's@\$ROLE_NAME@'${ROLE_NAME}'@g' ./common-tasks/templates/parameter-deny-all.json | jq .)
```

Comment:
Sorry for the very silly question: I got really confused by the `@` used as delimiter, I usually see `/` used as `sed` delimiter, e.g.

```
sed 's/\$ROLE_NAME/'${ROLE_NAME}'/g'
```